server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -153,7 +153,10 @@ public enum MetadataVersion {
     IBP_3_2_IV0(4, "3.2", "IV0", false),
 
     // Support for metadata.version feature flag (KIP-778)
-    IBP_3_3_IV0(5, "3.3", "IV0", false);
+    IBP_3_3_IV0(5, "3.3", "IV0", false),
+
+    // In KRaft mode, use BrokerRegistrationChangeRecord instead of UnfenceBrokerRecord and FenceBrokerRecord.
+    IBP_3_3_IV1(6, "3.3", "IV0", false);
```

Comment:
It's recommended to add corresponding `MetadataVersionTest` after adding new IBP